Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -30,6 +30,104 @@ public typealias ContainerViewController = UIViewController & NavigationComponen
  [Pricing Guide](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/).
  */
 open class NavigationViewController: UIViewController, NavigationStatusPresenter, NavigationViewData {
+    
+    // MARK: Accessing NavigationView
```

Comment:
It isn’t clear at a glance what NavigationView does or why one would access it:

```suggestion
    // MARK: Accessing the View Hierarchy
```